src/main/java/use_cases/add_edit_expenses_use_case/ExpenseIB.java

```diff
@@ -1,24 +1,23 @@
-package use_case.add_edit_expenses_use_case;
+package use_cases.add_edit_expenses_use_case;
 
 import entities.EntityException;
 
 /**
  * ExpenseIB passes in user input of expense information in the form of ExpenseID.
+ * Implemented by ExpenseUCI.
  */
 
 public interface ExpenseIB {
     /**
-     * Returns String success message in the form of a ExpenseOD object when an expense is successfully added,
-     * returns String fail messages that are context specific when failed.
+     * Describes abstract addExpenseInMonth() that is override in ExpenseUCI to format detailed responses for attempts to add an Expense.
```

Comment:
Sorry for being so nitpicky with this but maybe you could write "Adds an expense to the current month and returns a response describing if it was added successfully or not."

Reply:
And same goes for the edit version below.